doc/classes/FileAccess.xml

```diff
@@ -153,6 +153,20 @@
 				Returns the last error that happened when trying to perform operations. Compare with the [code]ERR_FILE_*[/code] constants from [enum Error].
 			</description>
 		</method>
+		<method name="get_file_as_array" qualifiers="static">
+			<return type="PackedByteArray" />
+			<param index="0" name="path" type="String" />
+			<description>
+				Returns the whole [param path] file contents as a [PackedByteArray]
```

Comment:
```suggestion
				Returns the whole [param path] file contents as a [PackedByteArray].
```

Reply:
Maybe should also clarify the encoding?

Reply:
Encoding? It's reading the file as is (it's for primary for reading binary files, not text).

Reply:
Ah right, makes sense.

Reply:
Added `without any decoding.` to clarify it's reading raw bytes.